server/src/test/java/org/elasticsearch/search/lookup/LeafDocLookupTests.java

```diff
@@ -72,4 +73,10 @@ public void testLookupWithFieldAlias() {
         ScriptDocValues<?> fetchedDocValues = docLookup.get("alias");
         assertEquals(docValues, fetchedDocValues);
     }
+
+    public void testTypesDeprecation() {
+        ScriptDocValues<?> fetchedDocValues = docLookup.get("_type");
+        assertEquals(docValues, fetchedDocValues);
+        assertWarnings("[types removal] Looking up doc types in scripts is deprecated.");
```

Comment:
If this warning will be written out, then no need for a package private constant in leaf doc lookup.

Reply:
Fixed.